curriculum/challenges/english/25-front-end-development/workshop-pin-extractor/68557bd29f7f3c1880f98836.md

```diff
@@ -0,0 +1,79 @@
+---
+id: 68557bd29f7f3c1880f98836
+title: Step 8
+challengeType: 20
+dashedName: step-8
+---
+
+# --description--
+
```

Comment:
This step might be too much (maybe not). Probably campers should be guided a bit more.

Reply:
Would you say that it should separate into steps, or have more explit instructions?